packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -279,19 +280,32 @@ class Opacity extends SingleChildRenderObjectWidget {
   /// would otherwise contribute relevant semantics.
   final bool alwaysIncludeSemantics;
 
+  /// Whether the child render object is always be painted.
+  ///
+  /// Defaults to false. The child render object do not be painted when
+  /// [opacity] is 0.0.
+  ///
+  /// When true, regardless of the opacity settings the child render object is
+  /// always be painted as if it is visible. This is useful in cases some
+  /// render object such as [RenderParagraph] could perform some layout
+  /// processes during the painting phase.
```

Comment:
I'm curious about this - can you give more context on what exactly goes wrong? Perhaps if that causes bugs this should be prohibited instead?

Reply:
Coming soon!

Reply:
Hey, I have updated the context!

Reply:
Doing the layout in the painting phase sounds like a source of troubles. It looks like we should figure out how to avoid layout if only the color changes

Reply:
cc @LongCatIsLooong